includes/abstracts/abstract-wc-order.php

```diff
@@ -145,6 +145,41 @@ public function remove_order_items( $type = null ) {
 	}
 
 	/**
+	 * Returns a list of all payment tokens associated with the current order
+	 *
+	 * @since 2.6
+	 * @return array An array of payment token objects
+	 */
+	public function get_payment_tokens() {
+		return WC_Payment_Tokens::get_order_tokens( $this->id );
+	}
+
+	/**
+	 * Add a payment token to an order
+	 *
+	 * @since 2.6
+	 * @param  int                $token_id  ID for the token we are adding to the order
+	 * @param  WC_Payment_Token   $token     Payment token object
+	 * @return boolean True if the token was added, false if not
+	 */
+	public function add_payment_token( $token_id, $token ) {
```

Comment:
Does this method need both the `$token_id` and a `$token`params?

The `WC_Payment_Token` class has a `get_id()` method, so `$token->get_id()` could be used instead of requiring a `$token_id` param. :smile: